docs/spectrum1d.rst

```diff
@@ -87,6 +87,18 @@ A full list of the supported formats is shown in the table below.
 
 | More information on creating custom loaders can be found in the :doc:`custom loading </custom_loading>` page.
 
+Writing to a File
+-----------------
+
+Similarly, a `~specutils.Spectrum1D` object can be saved to any file format supported by
+:class:`~astropy.nddata.CCDData` via the :meth:`specutils.Spectrum1D.write` method.
```

Comment:
Why is CCDData involved here?

https://github.com/astropy/specutils/blob/25cb121f8da53a7e79ebc10b94f665e28e0399d4/specutils/spectra/spectrum1d.py#L20

Reply:
It comes from [NDIOMixin](https://docs.astropy.org/en/stable/api/astropy.nddata.NDIOMixin.html) whose API docs state CCDData 🤷

Reply:
I think NDIOMixin used CCDData as an example of implementation using the mixin. I don't think specutils has anything to do with CCDData.

Reply:
ok, would you rather that I just point directly to NDIOMixin then, or nothing at all and just leave the link to the API docs which inherits from NDIOMixin API?

Reply:
Why not link to https://specutils.readthedocs.io/en/latest/spectrum1d.html#reading-from-a-file (with `ref`)?

Reply:
that is _immediately_ before this sub-section, so I'll just update the text to mention "supported formats".